### PR TITLE
Added include for /etc/rsyslog.d/*.conf config files

### PR DESCRIPTION
**Pull request by zerkms**
Close https://github.com/wader/postfix-relay/issues/58

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ environment:
   - RSYSLOG_REMOTE_TEMPLATE=RSYSLOG_ForwardFormat
 ```
 
+#### Advanced logging configuration
+
+If configuration via environment variables is not flexible enough it's possible to configure rsyslog directly: `.conf` files in the `/etc/rsyslog.d` will be [sorted alphabetically](https://www.rsyslog.com/doc/v8-stable/rainerscript/include.html#file) and included into the primary configuration.
+
```
Comment (wader) [inline README.md:103]: maybe "...the `/etc/rsyslog.d` _directory_ will be ..."

**Comment by wader** [comment]
LGTM, maybe fix the doc thing then i'm ready to merge if you are

File: README.md (hunk 1 above)
Comment (zerkms) [reply README.md:103]: It was there, has been lost during multiple editing iterations. Thanks, fix comes in a second.

**Comment by zerkms** [comment]
There you go :rocket:

**Comment by wader** [comment]
Thanks 🥳

**Comment by hemberger** [comment]
There is already a file `/etc/rsyslog.d/postfix.conf` in the image:
```
# Create an additional socket in postfix's chroot in order not to break
# mail logging when rsyslog is restarted.  If the directory is missing,
# rsyslog will silently skip creating the socket.
$AddUnixListenSocket /var/spool/postfix/dev/log
```
This will now be included in a vanilla build. The directory `/var/spool/postfix/dev/log` does _not_ exist in the image, so it looks like this will do nothing (i.e. same behavior in vanilla build as before this change), but I wanted to point it out in case it's a relevant detail to anyone doing more customization.

**Comment by zerkms** [comment]
@hemberger 

```
# stat /var/spool/postfix/dev/log
  File: /var/spool/postfix/dev/log
  Size: 0         	Blocks: 0          IO Block: 1024   socket
Device: 10fc40h/1113152d	Inode: 43          Links: 1
Access: (0666/srw-rw-rw-)  Uid: (    0/    root)   Gid: (  102/ postfix)
Access: 2022-08-29 22:17:04.000000000 +0000
Modify: 2022-08-29 22:17:04.000000000 +0000
Change: 2022-08-29 22:17:04.000000000 +0000
 Birth: -
```

**Comment by wader** [comment]
@hemberger Thanks for the info, i did try a build/run and didn't see any error at least

**Comment by zerkms** [comment]
Btw, I run with include for several days no - no errors on start and overall it runs as great as before.

**Comment by hemberger** [comment]
Ah, I understand the documentation in that file now: `/var/spool/postfix/dev` is the directory they're referring to. So with this change, we are creating an additional socket in the vanilla build.

**Comment by hemberger** [comment]
I'm not an expert, but I don't think it's a problem -- probably just an imperceptible performance difference. :)

**Comment by wader** [comment]
Ok! let's leave it for now.

One thing i've thought about is some kind of test setup but it is a bit messy. Some kind of docker-compose thingy with a dns server (dnsmasq i've used before for this) and some kind of test program to send/receive? No something i will do anytime soon but maybe add an issue with help wanted?

**Comment by wader** [comment]
@hemberger Thanks for reviewing and keeping an eye!

**Comment by zerkms** [comment]
It's absolutely harmless: if postfix (or any other process) writes into that socket - it will be handled (and written to stdout) by rsyslog. It will be absolutely no runtime performance penalty to have there and possibly up to couple kilobytes memory footprint (may be less).

**Pull request merged**